docs/manual/developer/01_introduction.md

```diff
@@ -77,7 +77,23 @@ Those areas are going to be segmented by
 
 - Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
 - Area of effect: All PSPs that don’t have specific stakeholders, associated content and test scenarios.
-- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy paste, and also don’t change existing behavior for everybody (link to coding guidelines needed).
+- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy and paste, and also don’t change existing behavior for everybody.
+  See the [style guide](04_style_guide.md) for more details.
+
+##### Removal of Products
+Products that do not have any contributions for two years may be subject to removal.
```

Comment:
```suggestion
Products
  - that are past their declared EOL date, or
  - that don't have contributions to files in their respective product folder for two years may be subject to removal.
```

I suggest this formulation, so it is easier to track the two-years period, as one can add a prodtype to a rule, and that could also qualify as a "contribution". But how do you detect that? It is probably reasonable to expect that a product needs gradual profile updates if it is relevant.